### PR TITLE
Surface Uniswap V2 LP tokens

### PR DESCRIPTION
**Pull request by jinchung**
This PR is to surface both Uniswap V1 and V2 liquidity pool tokens in the investments section.
V2 pools when involving ETH are actually all WETH based - I am showing WETH tokens as ETH when displaying the V2 pool data.

The order of the tokens for V2 are determined by the "sort order" from the Uniswap SDK (which compares contract addresses with each other.) The order of the tokens for V1 are to always have the ETH token be displayed second.

To help simplify the logic for when Zerion or fallback data is used, I parse out the token "type" for Uniswap LP tokens.

### CODE REVIEW

File: src/redux/data.js [hunk 1]
```diff
@@ -261,10 +261,7 @@ export const addressAssetsReceived = (
   const liquidityTokens = remove(
     assets,
     asset =>
-      asset?.asset?.type === 'uniswap' ||
-      // Remove duplicate liquidity tokens when falling back from zerion
-      (toLower(asset?.asset.name).indexOf('uniswap') !== -1 &&
-        toLower(asset?.asset.name) !== 'uniswap')
+      asset?.asset?.type === 'uniswap' || asset?.asset?.type === 'uniswap-v2'
```
Comment (jinchung) [inline src/redux/data.js:264]: Just a note: I removed the extra check here for the fallback from Zerion bc now I parse out and include the uniswap LP token type in the fallback explorer discover tokens logic

File: src/components/coin-icon/CoinIconGroup.js [hunk 2]
```diff
@@ -46,15 +46,13 @@ export default function CoinIconGroup({ containerStyles, tokens }) {
     <Container css={containerStyles} shouldCenter={sizesTable[size].breakIndex}>
       {slicedArray.map((setOfTokens, lineIndex) => (
         <LineWrapper key={`coinLine_${lineIndex}`}>
-          {setOfTokens.map((token, index) => (
+          {setOfTokens.map((tokenSymbol, index) => (
             <Wrapper
               key={`coin_${index}_${lineIndex}`}
               width={sizesTable[size].width}
+              zIndex={-index}
```
Comment (brunobar79) [inline src/components/coin-icon/CoinIconGroup.js:53]: should you use map from `lodash` instead?

File: src/components/pools/PoolsListWrapper.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ export default function PoolsListWrapper({ data, totalValue = '0' }) {
         isVisible={!isInvestmentCardsOpen}
         pointerEvents={isInvestmentCardsOpen ? 'auto' : 'none'}
       >
-        {data.map(renderSavingsListRow)}
+        {data.map(renderInvestmentsListRow)}
```
Comment (brunobar79) [inline src/components/pools/PoolsListWrapper.js:33]: same

File: src/handlers/uniswapLiquidity.ts [hunk 4]
```diff
@@ -14,130 +15,303 @@ import { UNISWAP_V1_EXCHANGE_ABI } from '../references/uniswap';
...
+  quantity: string;
+}
+
+const getAssetCode = (token: LPToken): string => get(token, 'asset.asset_code');
+
+const getTokenDetails = async (
+  chainId: ChainId,
+  tokenAddress: string,
+  pairs: Record<string, SwapCurrency>
+) => {
```
Comment (brunobar79) [inline src/handlers/uniswapLiquidity.ts:57]: You might wanna create a new type TokenDetails and set it as the return type here.

**Review by brunobar79 — APPROVED** [review]
lit

**Review by skylarbarrera — APPROVED** [review]
🍳  LGTM, lmk if you want me to take another look after Bruno's suggestions!

**Pull request merged**